tox.ini

```diff
@@ -1,8 +1,35 @@
 [tox]
-envlist = py27-django18
+envlist = py{27,34,35,36}-django{18,19,110,111}, flake8
```

Comment:
See other comment about possibly dropping tests for currently unsupported Python/Django versions.